docs/UserGuide.md

```diff
@@ -40,6 +40,25 @@ Wondering what each icon is used for? You can refer to the table below to find o
 
 All commands and their examples are demarcated with `markups`. `Markups` appear as a grey box as shown.
 
+<div markdown="block" class="alert alert-info">
+
+**:clipboard: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. in `add n/NAME_OF_ASSIGNMENT`, `NAME_OF_ASSIGNMENT` is a parameter which can be used as `add n/Assignment 2`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME_OF_ASSIGNMENT [remind]` can be used as `n/Assignment 2 remind` or as `n/Assignment 2`.
+
```

Comment:
Extra line here. But i think once the UG is finalised then we can fix the spacings together